src/migrations/20220311135140-update-topics-table.js

```diff
@@ -0,0 +1,68 @@
+const TOPICS = [
+  'Behavioral / Mental Health / Trauma',
+  'Child Assessment, Development, Screening',
+  'CLASS: Classroom Organization',
+  'CLASS: Emotional Support',
+  'CLASS: Instructional Support',
+  'Coaching',
+  'Communication',
+  'Community and Self-Assessment',
+  'Culture & Language',
+  'Curriculum (Instructional or Parenting)',
+  'Data and Evaluation',
+  'ERSEA',
+  'Environmental Health and Safety / EPRR',
+  'Equity',
+  'Facilities',
+  'Family Support Services',
+  'Fiscal / Budget',
+  'Five-Year Grant',
+  'Home Visiting',
+  'Human Resources',
+  'Leadership / Governance',
+  'Learning Environments',
+  'Nutrition',
+  'Oral Health',
+  'Parent and Family Engagement',
+  'Partnerships and Community Engagement',
+  'Physical Health and Screenings',
+  'Pregnancy Services / Expectant Families',
+  'Program Planning and Services',
+  'Quality Improvement Plan / QIP',
+  'Recordkeeping and Reporting',
+  'Safety Practices',
+  'Staff Wellness',
+  'Teaching Practices / Teacher-Child Interactions',
+  'Technology and Information Systems',
+  'Transition Practices',
+  'Transportation',
```

Comment:
Now that these are being populated into the database, can we remove them from frontend/src/Constants.js and/or frontend/src/pages/ActivityReport/constants.js and/or src/constants.js and just pull the values from the DB?

Would we also need a front end under admin to allow adding/editing the list?

Reply:
I think these are both separate Jiras, which I will write

Reply:
see 
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-709
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-710